packages/bits/README.md

```diff
@@ -29,15 +29,15 @@ ng --version
 If you want to install the latest Angular CLI globally, run the following command:
 
 ```sh
-npm install -g @angular/cli@^12
+npm install -g @angular/cli
```

Comment:
Might want to leave this at a specific version in case angular updates and it is out of sync

Reply:
make sense